### ISSUE TITLE
PolySelectTool area remains the same at all zoom levels

### ISSUE THREAD

**Issue by respatialized** [bokeh/bokeh#5829]
When trying to select a series of points using the PolySelectTool, the area selected doesn't respect the zoom level of the plot. Particularly for the mapping application I'm working on, this is extremely undesirable behavior, as it precludes precise selection of points.

I first mentioned this over on the Google Group [here](https://groups.google.com/a/continuum.io/forum/#!topic/bokeh/m3JzMUFub9w). I was asked to open an issue here for the bug. Example screenshots from that post: 
![image](https://cloud.githubusercontent.com/assets/8691924/22471902/e6e492f8-e7a1-11e6-89e8-a5c8ee3fc457.png)
![image](https://cloud.githubusercontent.com/assets/8691924/22471909/ed113622-e7a1-11e6-8f74-f482c38a5ce8.png)

Here's the minimal code to reproduce the bug (area/points displayed differ from screenshots above). The two lines can be uncommented to show the issue on a tile map.

```python
from bokeh.io import (
    show,
    output_file
)
from bokeh.models import (
    PolySelectTool,
    ColumnDataSource,
    Circle,
    WheelZoomTool,
    PanTool
)
from bokeh.plotting import figure
#from bokeh.tile_providers import CARTODBPOSITRON
source = ColumnDataSource(
    {'coordsX':[-8215213.30496,
                -8173542.15792,
                -8207605.64239],
     'coordsY':[5265013.23126,
                5198976.24299,
                5267118.16664]})
p = figure(tools=["wheel_zoom","pan","poly_select"])
p.circle(x='coordsX',y='coordsY',source=source)
#p.add_tile(CARTODBPOSITRON)
output_file('poly-bug.html')
show(p)
```

Running Bokeh 0.12.4 on Linux Mint / Ubuntu inside a virtualenv. Let me know if I need to provide further information.

**Comment by bryevdv** [comment]
This should be fixed but moving out to `0.12.7`

**Comment by mattpap** [comment]
This should be fixed as a part of #12074 (in PR #12083).
